.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "cSpell.words": [
```

Comment:
this seems misplaced?

Reply:
It's for vscode spell checker. I can add this to ignore file.

Reply:
yeah I'm not sure this should go into the repo